eduhelx_jupyterlab_prof/handlers.py

```diff
@@ -465,14 +472,15 @@ async def set_root_folder_permissions(context: AppContext) -> None:
     # execute(["chmod", "a-w", repo_root.parent])
     ...
 
-async def sync_upstream_repository(context: AppContext) -> None:
-    course = await context.api.get_course()
+async def sync_upstream_repository(context: AppContext, course) -> None:
+    assignments = await context.api.get_my_assignments()
```

Comment:
I don't think `assignments` gets used in this method

Reply:
it does, for overwritable paths